src/Core/Resolvers/MsSqlQueryExecutor.cs

```diff
@@ -94,6 +109,19 @@ public MsSqlQueryExecutor(
             }
         }
 
+        /// <summary>
+        /// Function registered for callback during a hot-reload scenario.
+        /// </summary>
+        /// <param name="sender">The calling object.</param>
+        /// <param name="args">Event arguments.</param>
+        public void MsSqlQueryExecutor_ConfigChangeEventReceived(object? sender, CustomEventArgs args)
```

Comment:
Adding the prefix `MsSqlQueryExecutor_` to this function name add unnecessary redundancy. `MsSqlQueryExecutor` is already implied due to the class name.

Reply:
That's true, but we need a way to make the callbacks disambiguous between the many different QueryExecutors.